odoo/tools/mail.py

```diff
@@ -185,6 +185,11 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
 
     logger = logging.getLogger(__name__ + '.html_sanitize')
 
+    # Convert mso conditionals to mako tags, otherwise they are stripped by the
+    # cleaner. Restore them afterwards.
+    src = src.replace(u'<!--[if mso]>', u'<%mso_if%>')
```

Comment:
As we are about to remove support of mako tags with @Goaman  isn't it going to be broken once we remove the mako exceptions ?